introduction.md

```diff
@@ -12,4 +12,5 @@ The weapons I am profissioned with, are
 * c#
```

Comment:
Sure I can, if you really think that is something I should advertise.
I mean, seems a bit dangerous to advertise that I know the dark arts, don't you agree?